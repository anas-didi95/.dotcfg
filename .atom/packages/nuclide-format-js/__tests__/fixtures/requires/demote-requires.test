var one = require('one');

var two: ?one = null;
