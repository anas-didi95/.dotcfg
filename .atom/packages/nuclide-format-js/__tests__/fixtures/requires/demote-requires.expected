import type one from 'one';

var two: ?one = null;
